Week06/README.md

```diff
@@ -15,6 +15,16 @@
 - [Print the Elements of a Linked List](https://www.hackerrank.com/challenges/print-the-elements-of-a-linked-list/problem)
 - [Delete a Node](https://www.hackerrank.com/challenges/delete-a-node-from-a-linked-list/problem)
 - [Merge two sorted linked lists](https://www.hackerrank.com/challenges/merge-two-sorted-linked-lists/problem)
+- [Reverse Nodes in k-Group](https://leetcode.com/problems/reverse-nodes-in-k-group/description/)
+        [Java Solution](https://github.com/rajat123456/General-Competitive-Programming-Questions/blob/master/LeetCode(Reverse%20Nodes%20in%20k-Group).java)
+- [Remove Duplicates from Sorted List](https://leetcode.com/problems/remove-duplicates-from-sorted-list/description/)
+        [Java Solution](https://github.com/rajat123456/General-Competitive-Programming-Questions/blob/master/LeetCode(Remove%20Duplicates%20from%20Sorted%20List).java)
```

Comment:
add '-' at the beginning, check the format pls.

- Question name
   - solution